### DISCUSSION TITLE
Does this support ArcGIS Enteprirse 11.1 ?

### DISCUSSION THREAD

**Discussion by CyientNam1** [Esri/arcgis-powershell-dsc#469] [Q&A] [answered]
Does this support ArcGIS Enteprirse 11.1 ?

**Comment by cameronkroeker** [comment, marked as answer]
> Does this support ArcGIS Enteprirse 11.1 ?

Hi @CyientNam1,

The latest ArcGIS Module v4.1.0 does support ArcGIS Enterprise 11.1.

https://github.com/Esri/arcgis-powershell-dsc/releases/tag/v4.1.0

Thanks,
Cameron .

**Reply by CyientNam1** [reply]
Thank you.
